### PR TITLE
trigger-enhancements-for-dt

### PR DESCRIPTION
**Pull request by wjcunningham7**
The purpose of this PR is to support Postgres as a database trigger

- [x] I have added the tests to cover my changes.
- [x] I have updated the documentation and CHANGELOG accordingly.
- [x] I have read the CONTRIBUTING document.

### CODE REVIEW

**Comment by santoshkumarradha** [comment]
Left over items 
- [ ] documentation 
- - adding in features page trigger and also api and how to 
- [ ] Tests

File: covalent/triggers/base.py
Comment (kessler-frost) [inline covalent/triggers/base.py:1]: We don't need the changes in this file.

File: Dockerfile
Comment (kessler-frost) [inline Dockerfile:1]: Don't need the changes in this file either.

Comment (ArunPsiog) [reply Dockerfile:1]: As a result of the reverse merge from the `develop` branch, these changes were pushed to this current branch. I didn't add any other changes to it. If you insist on removing those changes, we can remove them.

File: covalent/triggers/base.py
Comment (ArunPsiog) [reply covalent/triggers/base.py:1]: Changed due to the reverse merge from the `develop` branch. If you want to remove those changes, we could.

Comment (kessler-frost) [reply covalent/triggers/base.py:1]: Oh @ArunPsiog I think I might've caused confusion here, these are the changes that @wjcunningham7 made even before the reverse merge from develop. I am just mentioning them here as a reminder. Basically whatever changes were made in these files are irrelevant to the new feature introduction and hence we don't need them at all, in other words: the files `base.py` and `Dockerfile` should not be different than what they are in `develop` branch.

File: Dockerfile
Comment (kessler-frost) [reply Dockerfile:1]: Similar to the above comment ^.

File: covalent/triggers/base.py
Comment (ArunPsiog) [reply covalent/triggers/base.py:1]: My bad :sweat_smile:

File: Dockerfile
Comment (ArunPsiog) [reply Dockerfile:1]: My bad :sweat_smile:

File: covalent_dispatcher/_triggers_app/app.py [hunk 1]
```diff
@@ -80,6 +80,11 @@ def get_threadpool():
     return ThreadPoolExecutor()
 
 
+@router.get("/triggers/healthcheck")
+async def healthcheck(request: Request):
+    return {"status": "ok"}
```
[the remark this answers is not in the document]
Comment (ArunPsiog) [reply covalent_dispatcher/_triggers_app/app.py]: Hi @kessler-frost, I have made changes to populate the `health check` endpoint on the `covalent start --triggers-only`.

File: covalent/triggers/base.py
Comment (ArunPsiog) [reply covalent/triggers/base.py:1]: @kessler-frost, Replaced `base.py` and `Dockerfile` from develop.

File: Dockerfile
Comment (ArunPsiog) [reply Dockerfile:1]: Replaced Dockerfile from `develop`.

**Review by kessler-frost — DISMISSED** [review]

**Review by santoshkumarradha — APPROVED** [review]

**Pull request merged**